src/libs/ReportUtils.js

```diff
@@ -2455,6 +2495,31 @@ function buildOptimisticTaskReport(ownerAccountID, assigneeAccountID = 0, parent
     };
 }
 
+/**
+ * A helper method to create transaction thread
+ *
+ * @param {Object} reportAction - the parent IOU report action from which to create the thread
+ *
+ * @returns {Object}
+ */
+function buildTransactionThread(reportAction) {
+    const participantAccountIDs = _.uniq([currentUserAccountID, Number(reportAction.actorAccountID)]);
+    return buildOptimisticChatReport(
+        participantAccountIDs,
+        getTransactionReportName(reportAction),
+        '',
+        lodashGet(getReport(reportAction.reportID), 'policyID', CONST.POLICY.OWNER_EMAIL_FAKE),
```

Comment:
Hi! This line also introduced a bug: https://github.com/Expensify/App/issues/26748 solved by this PR: https://github.com/Expensify/App/pull/27285

Here are more details: https://github.com/Expensify/App/issues/26748#issuecomment-1711561680